firmware/robot2015/src-ctrl/drivers/mcp23017/mcp23017.cpp

```diff
@@ -0,0 +1,281 @@
+/*  MCP23017 library for Arduino

```

Comment:
I haven't seen this yet. I'll have time to look at it tonight. The control board does have an interrupt line from the ioexpander connected to it. It should be fine as long as that will work.